packages/x-components/src/x-modules/empathize/components/empathize.vue

```diff
@@ -83,9 +103,10 @@
      * @internal
      */
     @XOn(component => (component as Empathize).eventsToOpenEmpathize)
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     open(payload: unknown, metadata: WireMetadata): void {
-      this.changeOpenState(true, metadata);
+      if (!this.$refs.isEmptySlot) {
```

Comment:
Is there a reason to not use the calculated `this.isEmptySlot` instead of using a ref?
```suggestion
      if (!this.isEmptySlot) {
```

Reply:
Hi @alvarodE it was the first approach, but the tests failed. I thought that it was because the content didn't get updated (as v-show only was hiding it) and then the condition was always true. It makes sense with the comment you added below about the `isEmptySlot` type. Does it makes sense to you, or I'm losing something?

Reply:
Using the calculated `isEmptySlot` property over the `ref` results in a buggy behaviour. The `@XOn` listener is triggered before the `updated` lifecycle hook (it is bound to the `created` one), so you end up having to, for example, click twice the `search box`, in order for the `empathize` to finally open.

```suggestion
      if (this.$refs.noContent) {
```